[jsk_interactive_marker] build fail  on generating java message

```
Traceback (most recent call last):
  File "/opt/ros/indigo/share/genjava/cmake/../../../lib/genjava/genjava_gradle_project.py", line 14, in <module>
    genjava.main(sys.argv)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/genjava/genjava_main.py", line 82, in main
    gradle_project.create(args.package, args.output_dir)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/genjava/gradle_project.py", line 152, in create
    raise IOError("could not find %s on the ros package path" % msg_pkg_name)
IOError: could not find jsk_interactive_marker on the ros package path
Traceback (most recent call last):
  File "/opt/ros/indigo/share/genjava/cmake/../../../lib/genjava/genjava_gradle_project.py", line 14, in <module>
    genjava.main(sys.argv)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/genjava/genjava_main.py", line 82, in main
    gradle_project.create(args.package, args.output_dir)
  File "/opt/ros/indigo/lib/python2.7/dist-packages/genjava/gradle_project.py", line 152, in create
    raise IOError("could not find %s on the ros package path" % msg_pkg_name)
IOError: could not find jsk_interactive_marker on the ros package path
make[2]: *** [java/jsk_interactive_marker/build.gradle] Error 1
make[1]: *** [CMakeFiles/jsk_interactive_marker_generate_messages_java.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
make[2]: *** [java/jsk_interactive_marker/build.gradle] Error 1
make[1]: *** [CMakeFiles/jsk_interactive_marker_generate_messages_java_gradle.dir/all] Error 2
make: *** [all] Error 2
```
any idea how to fix this error?

--- a/jsk_interactive_markers/jsk_interactive_marker/CMakeLists.txt
+++ b/jsk_interactive_markers/jsk_interactive_marker/CMakeLists.txt
@@ -18,6 +18,7 @@ endif()
 
 
 find_package(catkin REQUIRED COMPONENTS
+  message_generation
   cmake_modules jsk_pcl_ros jsk_footstep_msgs geometry_msgs visualization_msgs
   dynamic_reconfigure jsk_rviz_plugins
   interactive_markers dynamic_tf_publisher tf_conversions eigen_conversions
@@ -134,7 +135,7 @@ generate_messages(
 
 catkin_package(
     DEPENDS TinyXML
-    CATKIN_DEPENDS  geometry_msgs jsk_footstep_msgs tf_conversions actionlib jsk_rviz_plugins jsk_recognition_msgs
+    CATKIN_DEPENDS  geometry_msgs jsk_footstep_msgs tf_conversions actionlib jsk_rviz_plugins jsk_recognition_msgs message_runtime
     INCLUDE_DIRS include # TODO include
     LIBRARIES jsk_interactive_marker # TODO
 )

--- a/jsk_interactive_markers/jsk_interactive_marker/package.xml
+++ b/jsk_interactive_markers/jsk_interactive_marker/package.xml
@@ -34,11 +34,13 @@
   <build_depend>roscpp</build_depend>
   <build_depend>roslib</build_depend>
   <build_depend>message_filters</build_depend>
+  <build_depend>message_generation</build_depend>
   <build_depend>jsk_rviz_plugins</build_depend>
   <build_depend>pr2eus_moveit</build_depend>
   <build_depend>jsk_recognition_msgs</build_depend>
   <build_depend>jsk_topic_tools</build_depend> 
   <build_depend>yaml-cpp</build_depend>
+  <run_depend>message_runtime</run_depend>
   <run_depend>roseus</run_depend>
   <run_depend>tinyxml</run_depend>
   <run_depend>geometry_msgs</run_depend>

